daemon/mgr/image.go

```diff
@@ -92,12 +93,34 @@ func (mgr *ImageManager) PullImage(pctx context.Context, image, tag string, out
 		return err
 	}
 
+	ctx, cancel = context.WithCancel(pctx)
+	imageConfig, err := mgr.client.GetImageConfig(ctx, image+":"+tag)
```

Comment:
I think `image+":"+tag` is a little bit bad. Could we change this to ref.String()? And we use reference to pass parameter.

Reply:
I am afraid that will change the `PullImage()` pass-in parameter for using ref.String() instead of `image+":"+tag`.  Maybe I can change this in a follow-up PR if needed. @allencloud